app-accessibility/flite/patches/flite-2.1.patchset

```diff
@@ -0,0 +1,44 @@
+From fb6af52e85a36505056b10ca04647e5aacdfd36a Mon Sep 17 00:00:00 2001
+From: Timothy Gu <timothygu99@gmail.com>
+Date: Wed, 24 Dec 2014 22:37:15 +0000
+Subject: Support shared libraries on Haiku
+
+
+diff --git a/configure.in b/configure.in
+index 54a782d..65df51b 100644
+--- a/configure.in
++++ b/configure.in
+@@ -266,6 +266,10 @@ fi
+ 
+ if test "$shared" = true; then
+    case "$target_os" in
++	haiku*)
++		SHFLAGS="-fPIC"
++		OTHERLIBS="-lnetwork"
++	;;
+ 	linux*|k*bsd*-gnu|gnu*)
+ 		SHFLAGS="-fPIC"
+         ;;
+-- 
+2.19.0
+
+
+From b308bc70ba83533b48812ccc987046f77222f942 Mon Sep 17 00:00:00 2001
+From: Timothy Gu <timothygu99@gmail.com>
+Date: Wed, 24 Dec 2014 23:44:02 +0000
+Subject: Bump shlib version to libflite.so.2
+
+
+diff --git a/config/project.mak b/config/project.mak
+index 1d655d9..a1d0729 100644
+--- a/config/project.mak
++++ b/config/project.mak
+@@ -40,4 +40,4 @@ PROJECT_VERSION = 2.1
+ PROJECT_DATE = Dec 2017
+ PROJECT_STATE = release
+ 
+-PROJECT_SHLIB_VERSION = 1
++PROJECT_SHLIB_VERSION = 2
```

Comment:
@korli this is done the first time too, looking at their github page I see they still use 1 here, shouldn't we keep it at 1 also then?

Reply:
In the previous patch there was one line explaining this. It was: "This way it is consistent with libflite.so.2.0.0"

Reply:
OK, see it now, build with default 1 gives libflite_*_.awb.so1 ... should be OK to keep then :)